producers-commissioning/src/main/java/uk/gov/moj/sdt/producers/comx/config/MBeanConfig.java

```diff
@@ -0,0 +1,53 @@
+package uk.gov.moj.sdt.producers.comx.config;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.jmx.export.MBeanExporter;
+import org.springframework.jmx.export.assembler.MethodNameBasedMBeanInfoAssembler;
+import org.springframework.jmx.support.RegistrationPolicy;
+import uk.gov.moj.sdt.utils.mbeans.SdtMetricsMBean;
+
+@Configuration
+@EnableAutoConfiguration
+public class MBeanConfig {
```

Comment:
The equivalent class in civil-sdt is called ProducersConfig.  Should we call this ProducersCommissioningConfig (or ProducersComxConfig) just to be consistent?

Reply:
renamed to ProducersComxConfig